dtm0/fop.c

```diff
@@ -122,6 +124,32 @@ int m0_dtm0_fop_init(void)
 	return 0;
 }
 
+/*
+  Allocates a fop.
+ */
+
+int m0_dtm0_fop_create(struct m0_rpc_session	 *session,
+		       enum m0_dtm0s_msg	  opmsg,
+		       struct m0_dtm0_tx_desc	 *txr,
+		       struct m0_fop		**out)
+{
+	struct dtm0_req_fop  *op;
+	struct m0_fop	     *fop;
+	int		      rc = 0;
+
+	*out = NULL;
+
+	fop = m0_fop_alloc_at(session, &dtm0_req_fop_fopt);
+	if (fop == NULL)
+		rc = -ENOMEM;
```

Comment:
Probably, it has to be `return M0_ERR(-ENOMEM);`

Reply:
Done